searxng/searxng
# [mod] simple theme: move initilization of document.searxng to searxng.min.js

## What does this PR do?

`searxng.head.min.js` focuses on browser features detection nothing more (to add / remove some CSS classes).
The parsing and evaluation of `searxng.head.min.js` blocks the browser rendering (since it is at the top of the page, the page is blank).
Ideally, we should find a way to remove this file.

## Why is this change important?

* We can add as many parameters in `searxng.min.js`.
* If some CSS trick are found, we can remove `searxng.head.min.js`.

For example: 
* `<noscript><link rel="stylesheet" href="searxng.nojs.min.css"></noscript>`
* About touch screen: https://ferie.medium.com/detect-a-touch-device-with-only-css-9f8e30fa1134

## How to test this PR locally?

* `make run`
* check the UI in a desktop browser.
* check the UI in a mobile browser with a  touch screen.

## Author's checklist



## Related issues

Related to https://github.com/searxng/searxng/pull/562
